maya/core.py

```diff
@@ -336,10 +336,15 @@ def slang_date(self):
         dt = self.datetime(naive=True, to_timezone=self.local_timezone)
         return humanize.naturaldate(dt)
 
-    def slang_time(self):
-        """"Returns human slang representation of time."""
-        dt = self.datetime(naive=True, to_timezone=self.local_timezone)
-        return humanize.naturaltime(dt)
+    def slang_time(self, locale="en"):
+        """"Returns human slang representation of time.
+
+        Keyword Arguments:
+            locale -- locale to translate to, e.g. 'fr' for french.
+                       (default: 'en' - English)
+        """
+        dt = self.datetime(to_timezone=self.local_timezone)
+        return pendulum.instance(dt).diff_for_humans(locale=locale)
```

Comment:
I've played around a little with this `diff_for_humans()` methods and am quite confused now. Maybe you have an explanation:

Assume my current local time is about:  `Wed, 16 May 2018 19:44:12 +0200` and I'm using the latest maya release:

```pycon
>>> n = maya.now()
>>> n.slang_time()
'7 seconds ago'
>>> # now with pendulum:
>>> import pendulum
>>> dt = n.datetime(naive=True, to_timezone=n.local_timezone)
>>> pendulum.instance(dt).diff_for_humans()
'1 hour from now'
```

What bugs me is that `1 hour from now` which is from pendulum. (btw. the same happens if there is a `tzinfo` in `dt`)
Any idea why?